api/EXT_blend_minmax.json

```diff
@@ -5,40 +5,40 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/EXT_blend_minmax",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "38"
           },
           "chrome_android": {
-            "version_added": true
+            "version_added": "38"
           },
           "edge": {
             "version_added": "17"
           },
           "firefox": {
-            "version_added": "33"
```

Comment:
This seems to have come from the initial migration in https://github.com/mdn/browser-compat-data/pull/70, so probably an error from the wiki.